spec/unit/worker_spec.rb

```diff
@@ -145,6 +145,12 @@ def around_perform(job)
         end
         worker.perform(job)
       end
+
+      it 'can save log trace' do
+        JobClass.stub(:perform)
+        worker.log_stack_trace
+        expect(log_output.string).to match(/unit\/worker_spec\.rb/)
```

Comment:
> Doesn't this regexp syntax allow you to use | as a delimiter?

You can use any delimiter you want by using `%r`:

``` ruby
%r{unit/worker\.spec}
%r|unit/worker\.spec|
```

It is nice to do that when you have lots of `/` characters in your expression but there's only one here so I don't think it matters much either way.